src/components/Hello.vue

```diff
@@ -1,7 +1,28 @@
 <template>
-  <div class="hello">
-    <h1>{{ msg }}</h1>
+
+  <div class="park">
+
+
+  
+    <p> I'm interested in joining the mPower study because I </p>
+    <br>
+    <br>
+    <select placeholder= '' id="interest">
```

Comment:
I think that for this select dropdown, and the one below, it would be cleaner to use a two-way data binding. 

https://vuejs.org/v2/guide/forms.html#Select